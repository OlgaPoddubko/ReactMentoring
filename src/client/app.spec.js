import React from 'react';
import App from './app';

describe('render App', () => {
  it('should render App', () => {
    const wrapper = shallow(<App />);
    expect(wrapper).toMatchSnapshot();
  });
});
