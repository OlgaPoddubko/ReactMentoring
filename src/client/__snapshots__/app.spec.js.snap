// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render App should render App 1`] = `
<div>
  <ul>
    <li>
      <Link
        replace={false}
        to="/"
      >
        Home
      </Link>
    </li>
    <li>
      <Link
        replace={false}
        to="/movie"
      >
        Movie
      </Link>
    </li>
  </ul>
  <Switch>
    <Route
      component={[Function]}
      exact={true}
      path="/"
    />
    <Route
      component={[Function]}
      path="/movie"
    />
    <Redirect
      push={false}
      to="/"
    />
  </Switch>
</div>
`;
